src/models/user.js

```diff
@@ -0,0 +1,15 @@
+import mongoose from 'mongoose'
+
+const userSchema = new mongoose.Schema({
+    firstName: String,
+    lastName: String,
+    mobile: String,
+    isMobileVerified: Boolean,
+    isActive: Boolean,
+    createdAt: Date,
```

Comment:
remove createdAt and updatedAt

instead add following object,

`{ timestamps: true}`